provision-contest/ansible/roles/ssh/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+github_ssh_user:
+  - eldering
+  - meisterT
+  - nickygerritsen
+  - thijskh
+  - tuupke
+  - ubergeek42
+  - vmcj
```

Comment:
Nitpick: here and in the ignore a newline is missing.

Reply:
The ansible linter doesn't want a newline for vars files, but I agree it looks better when its there.